### ISSUE TITLE
Deep Neural Networks for No-Reference and Full-Reference Image Quality Assessment

### ISSUE THREAD

**Issue by ghuijo** [yskim0/One-Day-One-Paper#71]
## Abstract
* deep neural network-based approach to IQA.
* end-to-end로 train됨.
* 10 conv layers & 5 pooling layers for feature extraction +  2 fc layers for regression
* NR과 FR setting에서 모두 사용 가능.
* local quality와 local weights의 joint learning이 가능.
* LIVE, CISQ, TID2013 databases, LIVE in the wild image quality challenge database로 평가함.
* NR, FR IQA methods에서 우수한 성능을 보여줌.

### Neural Network-Based FR IQA
![image](https://user-images.githubusercontent.com/67627471/129565918-a2e7951d-5238-4a6d-a71a-4550d5c68582.png)

* Siamese networks: learn similarity relations between two inputs.
→ inputs을 병렬로 놓아서 두 networks가 synaptic connection weights를 공유하도록 함.
→ Siamese network를 feature extraction에 이용.
* 추출된 features를 IQA의 regression problem에 쓰기 위해 feature extraction 다음 feature fusion step 과정 거침.
* The fused features는 regression 파트의 input이 됨.
* VGGnet을 기초로 사용. (3×3)
→ VGG network의 input은 224×224 pixels 사이즈의 이미지.
→ 32×32 pixel-sizes patches 정도의 더 작은 인풋에 맞추기 위해서 network를 3개의 layers(conv3-32, conv3-32, maxpool)로 확장함. (기존 네트워크 앞쪽에 둠.)
* 모든 conv layers는 3 × 3 pixel-size convolution kernels을 적용하고, ReLU activation function 사용됨.
* 인풋과 같은 크기의 아웃풋을 얻기 위해 conv에 제로패딩 적용함.
* 오버피팅 방지용으로 fc layers에 Dropout regularization(ratio 0.5) 적용함.

### Network adaptations for NR IQA
![image](https://user-images.githubusercontent.com/67627471/129566568-dc91cd7e-6e5b-446e-8447-85918c346afe.png)

* reference patch에 피쳐가 없기 때문에 feature pooling은 필요 없음.(no-reference)
* FR IQA와 loss function 동일.

### training
* backpropagation에 의해 반복적으로 train됨.
* In each epoch the training set is divided into mini-batches for batchwise optimization.
* For training the FR IQA networks, the respective reference patches are included in the mini-batch. Patches are randomly sampled every epoch to ensure that as many different image patches as possible are used in training.
* The learning rate for the batchwise optimization is controlled per parameter adaptively using the ADAM method based on the variance of the gradient.
* LIVE나 TID2013으로 train됨.

## Results
* LIVE와 TID2013 databases로 test한 결과.
<image src="https://user-images.githubusercontent.com/67627471/129570532-a8568d1f-81d9-45ab-b43a-601a6a924ec4.png" width="50%" height="50%" />
* 두 databases끼리 공통된 reference images가 많은 점 참고.
* TID2013을 여러 서브셋으로 나눠 성능 비교.
<image src="https://user-images.githubusercontent.com/67627471/129571106-f6dd1ac5-48d5-422b-b3e1-b26207263d9c.png" width="50%" height="50%" />
* NR IQA를 CLIVE dataset으로 평가.
<image src="https://user-images.githubusercontent.com/67627471/129571179-7109dd3f-c724-423e-9d24-4369fa7b78a3.png" width="40%" height="40%" />

##
* Siamese networks??
